MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -53,6 +56,8 @@ public override SpriteFontContent Process(FontDescription input, ContentProcesso
                         if (File.Exists(fontFile))
                             break;
                     }
+                    if (File.Exists(fontFile))
```

Comment:
no it isn't. We want to abort on the first file we find. Without this we will always end up picking up the last file we find (in this case the one in /Library/Fonts. 
That first break only breaks out of the inner loop. We want to break out of the top one too if we find a file.